### PR TITLE
Data import and export playbooks for dev-env

### PR DESCRIPTION
**Pull request by nathanielknight**
This PR adds two playbooks to the `dev-env` directory for importing and exporting dev data.

The `export-data.yml` playbook puts a database dump and a copy of the Kive MEDIA_ROOT in a TAR file and then downloads it.

The `import-data.yml` playbook uploads a TAR file created with the export playbook and applies it to the `dev-env` (loading the database dump and restoring the files from MEDIA_ROOT).

The import is destructive (i.e. any data in the database will be dropped before the import happens), so it is probably not suitable for use in a production environment.

Both playbooks can be run from the `dev-env` directory in the project's Vagrant box, which results in the archive file being stored in a mapped directory. Both playbooks can also be run from a separate host if Ansible (i.e. an inventory is supplied to replace the one in `dev-env`).

### CODE REVIEW

File: dev-env/export-data.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      become: true
+      archive:
+        dest: "{{ archive_file }}"
+        format: tar
+        path:
+          - "{{ kive_media_root }}/*"
+          - "{{ dbdump_file}}"
+        exclude_path:
+          - "{{kive_media_root}}/ContainerRuns"
+          - "{{kive_media_root}}/Sandboxes"
```
Comment (donkirkby) [inline dev-env/export-data.yml:32]: Sandboxes folder isn't used anymore.

Comment (donkirkby) [reply dev-env/export-data.yml:32]: Also, it looks like you're including the db dump in the tar file. If so, is it worth deleting the db dump file?

Comment (donkirkby) [reply dev-env/export-data.yml:32]: Oh, and does fetch leave a copy of the tar file behind? Maybe you should clean that up too.

Comment (nathanielknight) [reply dev-env/export-data.yml]: Done.

**Comment by nathanielknight** [comment]
I've updated the import playbook with a large warning message and a confirmation prompt. I've also ignored TAR files in the dev-env directory to avoid committing backups.

File: dev-env/import-data.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      register: confirm_delete
+      pause:
+        prompt: |-
+
+          ██╗    ██╗ █████╗ ██████╗ ███╗   ██╗██╗███╗   ██╗ ██████╗ ██╗
+          ██║    ██║██╔══██╗██╔══██╗████╗  ██║██║████╗  ██║██╔════╝ ██║
+          ██║ █╗ ██║███████║██████╔╝██╔██╗ ██║██║██╔██╗ ██║██║  ███╗██║
+          ██║███╗██║██╔══██║██╔══██╗██║╚██╗██║██║██║╚██╗██║██║   ██║╚═╝
+          ╚███╔███╔╝██║  ██║██║  ██║██║ ╚████║██║██║ ╚████║╚██████╔╝██╗
+          ╚══╝╚══╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚═╝  ╚═══╝╚═╝╚═╝  ╚═══╝ ╚═════╝ ╚═╝
```
Comment (donkirkby) [inline dev-env/import-data.yml:19]: :laughing: Subtle. I like it.

**Pull request merged**